### PR TITLE
Issue-802: Ensure commit is on graph

### PR DESCRIPTION
**Pull request by l-1squared**
If the bot does not have admin rights on github, the push to master will fail. However, the creation of the tag will be successful creating a tag object that is not on the graph. To avoid this we place the commit on a branch an push that branch, to be merged manually after the release

Signed-off-by: l-1sqared <30831153+l-1squared@users.noreply.github.com>

### CODE REVIEW

File: scripts/source_files/release_functions.sh [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
+  done
 }
 
-function releaseRepositoryAndPushVersion()
-{
+function push_version() {
+  echo Pushing version and tag to GitHub repository...
+  git push && push_result="success" || push_result="failure"
+  if [ push_result != "success" ]; then
+    branch_name="release/${VERSION_PREFIXED}"
```
Comment (fudler) [inline scripts/source_files/release_functions.sh:17]: Maybe it's too early in the morning, but where is the VERSION_PREFIXED defined? 
Further, at the current point the clean-up of this branch has to be done manually, doesn't it?

Comment (fudler) [reply scripts/source_files/release_functions.sh:17]: Ah, and this branch also has to be provided to other scripts somehow, I guess?

Comment (l-1squared) [reply scripts/source_files/release_functions.sh:17]: VERSION_PREFIXED is defined in the release.sh. Not the best design, I know...

Comment (l-1squared) [reply scripts/source_files/release_functions.sh:17]: > Ah, and this branch also has to be provided to other scripts somehow, I guess?

Good catch. I have not thought of that!

**Comment by l-1squared** [comment]
Since this solution is fragile and only mitigates the problem (we still have to merge manuall) we'll keep handing Admin rights to the release bot part of the process.

**Pull request closed without merging**